go/adbc/validation/validation.go

```diff
@@ -65,6 +65,8 @@ type DriverQuirks interface {
 	// have the driver drop a table with the correct SQL syntax
 	DropTable(adbc.Connection, string) error
 
+	ExampleSchema() string
```

Comment:
Should we call this `DbSchema` for consistency and so that it's not confused with an Arrow schema?

Reply:
fair, that's a good point